File: C++/Basics/Practice.cpp
#include <iostream>

using namespace std;



int evenodd(){
  int number;
  cout << "Enter a number: ";
  cin >> number;

  if(){
    cout << "The number is even. ";
  }
  else {
    cout << "The number is odd.";
  };
  return 0;
}

int primecomposite() {
    int num;
    int ans;

    cout << "Enter a number: ";
    cin >> num;
    for(int i = 0; i <= num; i++){
      ans = num / i;
    }

    if(ans > 2){
      cout << "The number is composite";
    }else if(ans == 2){
      cout << "The number is Prime";
    }

  return 0;
}

int factorial() {
  int num;
  count << "Enter a number: ";
  cin >> num;
  
  int count = 1;
  for(int i;i<num;i++){
    ans = count * i;
  }
}

int reverse(){
  int num = 1234;
  int reversed_num = 0;
  int count;

  while (num != 0){
    count = num % 10
    reversed_num = reversed_num * 10 + count;
    num = num / 10;
  }
  cout << "Reverse = " << reversed_num;
 
}

int max(){
  int num1;
  int num2;
  int num3;

  cout << "Input First Number: ";
  cin >> num1;
  cout << "";
  cin >> num2;
  cout << "";
  cin >> num3;
}

int main(){
  evenodd();
  primecomposite();
  factorial();
  reverse();
}

File: Python/Introduction/Practice.py
#Practice Questions:

#Factorial of a number
def factorial():
  x = input()
  factorial = 1
  for i in range(1,int(x)+1):
    factorial = 1 * factorial

  print(factorial)

def evenodd():
  x = input("Enter a number.")
  if x % 2 = 0:
    print("The number is even.")
  else 
    print("The number is odd.")

def primecomp():
  num = input("Enter a number: ")
  for i in range(1,int(num)+1):
    if num % i = 0:
      count = count + 1

  if count == 2:
    print("The number is prime.")
  else if count > 2:
    print("The number is composite.")

def reverse():
  num = 1234
  reversed_num = 0

  while num != 0:
    digit = num % 10
    reversed_num = reversed_num * 10 + digit
    num //= 10
    print("Reversed Number: " + str(reversed_num))

  def maximum():
    x = input("Enter First Number: ")
    y = input("Enter Second Number: ")
    z = input("Enter Third Number: ")

    if x > y: 
      if x > z:
        print("The largest number is: ",x)
      else
        print("The largest number is: ",z)
    else
      if y > z:
        print("The largest number is: ",y)
      else
        print("The largest number is: ",z)
  
#==
#Outputs
factorial()
evenodd()
primecomp()
